includes/Plugin.php

```diff
@@ -178,6 +187,9 @@ public function register() {
 		$this->admin = new Admin();
 		add_action( 'admin_init', [ $this->admin, 'init' ] );
 
+		$this->customizer = new Customizer();
```

Comment:
`$this->customizer` is not being used anywhere, so use local variable instead of property. In that way, we would not have to suppress PHPMD

Reply:
It is currently not used anywhere but it would be helpful to get by plugin instance like all other class instances.